### PR TITLE
feat: add 'by the Chrome DevRel'  floating ribbon

### LINKED ISSUE #9372: Move "by Chrome DevRel" branding out of logo
via @devnook. Instead of signaling that web.dev is a product maintained by the Chrome DevRel team in the logo, the logo needs to be reverted to its prior state. Hence we need a new place to convey the same brand message.

### PR DESCRIPTION
**Pull request by mamieorine**
Fixes #9372 

Changes proposed in this pull request:

- Changes the logo on the header
- Adds '**by the Chrome DevRel**' floating ribbon

### CODE REVIEW

**Review by rachelandrew — APPROVED** [review]
LGTM, but wait for Ewa's check.

File: src/site/_includes/partials/site-header.njk [hunk 1]
```diff
@@ -7,7 +7,7 @@
       <span class="tooltip__content" id="menu-button-toolip">Open menu</span>
     </button>
     <a href="/" class="site-header__brand brand">
-      {{ svg('../../images/logo.svg', {label: 'web.dev'}) }}
+      {{ svg('../../images/logo-without-devrel.svg', {label: 'web.dev'}) }}
```
Comment (matthiasrohmer) [inline src/site/_includes/partials/site-header.njk:10]: I think we should clean-up here. The logo with the "by Chrome DevRel" subline will never be used again, so remove it, rename `logo-without-devrel` to just `logo` and one ever needs to wonder why there is an unused one.

**Review by jimjiminyjimjim — COMMENTED** [review]
LGTM

File: src/site/_includes/partials/site-footer.njk [hunk 2]
```diff
@@ -1,73 +1,83 @@
...
-            data-category="Site-Wide Custom Events" data-label="Footer Link (index 2)">
-            {{ 'i18n.footer.view_source' | i18n(locale) }}
-          </a>
-        </li>
-      </ul>
+    <div class="site-footer__primary-nav-content">
+      <a href="/" class="site-footer__brand brand">
+        {{ svg('../../images/logo.svg', {label: 'web.dev'}) }}
+      </a>
+      <p class="text-size-0">We want to help you build beautiful, accessible, fast, and secure websites that work cross-browser, and for all of your users. This site is our home for content to help you on that journey, written by members of the Chrome team, and external experts.</p>
```
Comment (matthiasrohmer) [inline src/site/_includes/partials/site-footer.njk:9]: This needs to go through i18n, like for example `{{ 'i18n.footer.mission' | i18n(locale) }}`

**Review by matthiasrohmer — COMMENTED** [review]
One comment, and I am wondering about the logo: when looking at the Files Changed tab, the previous logo did not have `web.dev` written into it but was of the same size...?

Another note: please make sure to keep PRs distinct. The footer changes could have gone in a separate one.

File: src/site/_includes/partials/site-footer.njk (hunk 2 above)
Comment (mamieorine) [reply src/site/_includes/partials/site-footer.njk]: I have moved all footer changes to another PR: https://github.com/GoogleChrome/web.dev/pull/9520

**Comment by mamieorine** [comment]
> One comment, and I am wondering about the logo: when looking at the Files Changed tab, the previous logo did not have `web.dev` written into it but was of the same size...?
> 
> Another note: please make sure to keep PRs distinct. The footer changes could have gone in a separate one.

Actually, the previous logo also has words written on it but it's transparent so we cannot see it in the _**Files Changed**_ tab. For the new logo, I exported the SVG code from the Figma provided by Dan and it contains web.dev. with solid color.

I think the new logo's size is correct since the blue logo (before the text) has the same size as the old one. Also, the previous logo contains some extra space so the height is a bit higher than the current one.

**Review by devnook — APPROVED** [review]

**Pull request merged**